### PR TITLE
feat: Automation plugin complementary sidebar and fix for spreadsheet edge functions

### PR DESCRIPTION
**Pull request by richburdon**
TODO: Function isn't initially called.

Screenshot 2024-10-29 at 6 36 14 PM

### CODE REVIEW

File: packages/plugins/experimental/plugin-script/src/ScriptPlugin.tsx [hunk 1]
```diff
@@ -121,12 +121,11 @@ export const ScriptPlugin = (): PluginDefinition<ScriptPluginProvides> => {
...
 
-            // case 'complementary--settings': {
-            // return <div>Settings!</div>;
-            // }
-
             case 'complementary--automation': {
-              return <AutomationPanel subject={data.subject as any} />;
+              return {
+                node: <AutomationPanel subject={data.subject as any} />,
+                disposition: 'hoist',
```
Comment (wittjosiah) [inline packages/plugins/experimental/plugin-script/src/ScriptPlugin.tsx:127]: does this need to be hoisted?

File: packages/plugins/plugin-sheet/deprecated/Sheet/Sheet.tsx
Comment (wittjosiah) [inline packages/plugins/plugin-sheet/deprecated/Sheet/Sheet.tsx:1]: could this just be deleted now?

File: packages/plugins/plugin-sheet/src/compute-graph/compute-graph.ts [hunk 2]
```diff
@@ -130,7 +133,7 @@ export class ComputeGraph extends Resource {
 
   /**
    * Map bound value to custom function invocation.
-   * E.g., "HELLO(...args)" => "EDGE("HELLO", ...args)".
+   * E.g., "HELLO(...args)" => "DX("HELLO", ...args)".
```
Comment (wittjosiah) [inline packages/plugins/plugin-sheet/src/compute-graph/compute-graph.ts:136]: was this changed elsewhere?

**Review by wittjosiah — APPROVED** [review]

File: packages/plugins/experimental/plugin-script/src/ScriptPlugin.tsx (hunk 1 above)
Comment (richburdon) [reply packages/plugins/experimental/plugin-script/src/ScriptPlugin.tsx]: Need to discuss possibility of sorting these.

File: packages/plugins/plugin-sheet/deprecated/Sheet/Sheet.tsx
Comment (richburdon) [reply packages/plugins/plugin-sheet/deprecated/Sheet/Sheet.tsx:1]: Yes, TODO for @thure

File: packages/plugins/plugin-sheet/src/compute-graph/compute-graph.ts (hunk 2 above)
Comment (richburdon) [reply packages/plugins/plugin-sheet/src/compute-graph/compute-graph.ts:136]: Yes, and tested.

**Pull request merged**